docs-src/guide.scrbl

```diff
@@ -25,3 +25,4 @@ The following guides are available:
 @include-section["guide-limits.scrbl"]
 @include-section["guide-reach.scrbl"]
 @include-section["guide-editor-support.scrbl"]
+@include-section["guide-rpc.scrbl"]
```

Comment:
I think that this new content should live in the reference under a new section, 5.7 called "Participant Frontends". We'll move some of the content in 5.6.1 into 5.7.1 and then the RPC stuff will be in 5.7.2. For now, just make 5.7 and put the RPC stuff under there.